modules/manage/pages/manage-expiration.adoc

```diff
@@ -1,6 +1,156 @@
+----
+UPDATE user_sessions
+    USE KEYS "123456"
+    SET cart = [{"item": "9999", "quantity": "3"}, {"item" : "3423", "quantity" : 1}];
+----
+
+If you then query the document's expiration metadata, you'll find that it's now 0:
+
+[source, json]
+----
+[
+  {
+    "expiration": 0
+  }
+]
+----
+
+You can prevent Couchbase Server from clearing the expiration value by using the xref:settings:query-settings.html#preserve_expiry[`prevent_expiry`] request-level parameter. 
+
+You can also set the expiration in a mutation based on the current expiration value. 
+For example, you could alter the previous example statement to preserve the existing expiration value:
+
+[source, sql++]
+----
+UPDATE user_sessions AS u
+    USE KEYS "123456"
+    SET cart = [{"item": "9999", "quantity": "3"}, {"item" : "3423", "quantity" : 2}],
+        meta(u).expiration = meta(u).expiration;
+----
+
+== Have Buckets and Collections Automatically Expire Documents
+
+By default, documents expire only if you explicitly set their expiration values.
+You can have collections and buckets set their documents to automatically expire by changing their `maxTTL` setting. 
+Setting `maxTTL` to a non-zero value has two effects:
+
+* You cannot explicitly set a document's expiration to be longer than the `maxTTL` setting. 
+If you try to set the expiration to a value larger than the collection or bucket's `maxTTL` setting, Couchbase Server uses the `maxTTL` value instead.
+
+* Couchbase Server sets a default expiration for all documents you create or mutate in the collection or bucket unless you explicitly set the expiration to a smaller value.
+
+You can change the `maxTTL` setting for buckets and collections several ways. 
+
+=== Set maxTTL Using Couchbase Server Web Console
+
+When you create a new bucket or collection using the Web Console, you can enable the `maxTTL` setting.
+
+To enable automatically expiring documents when creating a bucket using the Web Console:
+
+. In the *Add Data Bucket* dialog, expand the *Advanced bucket settings* section.
+. Under *Bucket Max Time-To-Live* select *Enable*.
+. In the text field, enter the default TTL for documents in seconds.
+
+
+To enable expiration when creating a collection using the Web Console, enter a non-zero value in the *Add Collection to _scope name_ Scope* dialog's *Collection Max Time-To-Live* field.
+
+You can also use the Web Console to change the `maxTTL` setting of an existing bucket:
+
+. Click btn:[Buckets].
+. Click the name of the bucket whose `maxTTL` setting you want to change.
+. Click btn:[Edit].
+. Expand the *Advanced bucket settings* section.
+. Under *Bucket Max Time-To-Live*, select or clear the *Enable* box to enable or turn off automatic expiration.
+, If you're enabling automatic expiration, enter the number of seconds the documents should exist before expiring in the text field.
+. Click btn:[Save Changes].
+
+You cannot use the Web Console to change the `maxTTL` of a collection after ypu create it.
```

Comment:
The UI does support changing maxTTL.  Go to "Cluster>Buckets>Scopes & Collections" and expand to see the collections.  Click on "edit TTL" to make the change.

Reply:
I'll keep that in mind when I merge to 7.6. This PR is for the version 7.0 documentation. The statement about collections not being editable should stand for versions 7.0-7.2?

Reply:
>  The statement about collections not being editable should stand for versions 7.0-7.2?

Yes, the support for changing the maxTTL for a collection is in 7.6 via MB-31806

Reply:
Hm. That ticket fell through the cracks... I've contacted Beth and she'll made a document ticket to cover that.